src/components/Task.stories.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 import { withKnobs, object } from '@storybook/addon-knobs/react';
 
 import Task from './Task';
 
+export default {
+  Component: Task,
+  title: 'Task',
+  decorators: [withKnobs],
+  excludeStories: /^[a-z].*/,
```

Comment:
So I realised a convention that stories are PascalCased and data is camelCased would work well. What do you think @shilman?

Also super happy to do the `xData` thing if that seems too tricky (I'm kind of on the fence TBH).

We need a way to do `excludeStories` in `.storybook/config`, but this is OK for now.

Reply:
I prefer `xData`. Too many ways to shoot yourself in the foot with this way.

Reply:
OK, will update.